recipes/igdiscover/meta.yaml

```diff
@@ -24,7 +24,7 @@ requirements:
     - snakemake >=3.9.0
     - cutadapt
     - xopen >=0.1.1
-    - seaborn >=0.6.0
+    - seaborn =0.7
```

Comment:
If you pin to a specific version you restrict the possible environments that can be created. Just imagine a package that needs <=0.6. This package would not be compatible with your package here.